src/modal/modal.component.ts

```diff
@@ -217,7 +217,7 @@ export class Modal implements AfterViewInit, OnChanges {
 	get shouldShowScrollbar() {
 		const modalContent = this.modal ? this.modal.nativeElement.querySelector(".bx--modal-content") : null;
 		if (modalContent) {
-			const modalContentHeight = modalContent.getBoundingClientRect().height;
+			const modalContentHeight = Math.round(modalContent.getBoundingClientRect().height); // get rounded value from height to match integer returned from scrollHeight
```

Comment:
```suggestion
			// get rounded value from height to match integer returned from scrollHeight
			const modalContentHeight = Math.round(modalContent.getBoundingClientRect().height);
```

gotta do it in separate lines cause the line gets too long otherwise and linter complains.

Reply:
Kudos for figuring it out, tho, this one appears super randomly and I was never able to replicate it.